Source/WebCore/rendering/svg/RenderSVGInlineText.cpp

```diff
@@ -89,7 +89,8 @@ String RenderSVGInlineText::originalText() const
 
 void RenderSVGInlineText::setRenderedText(const String& text)
 {
-    RenderText::setRenderedText(text);
+    auto adjustText = applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre);
+    RenderText::setRenderedText(adjustText);
```

Comment:
It's a bit sad that we can't move the value into `setRenderedText()`. We should add an overload.